docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -13,5 +13,35 @@ There you can find links to upgrade notes for other versions too.
 
     **Be careful, this upgrade will remove sessions**
 
+### Application
+ - start filter products with elasticsearch ([#943](https://github.com/shopsys/shopsys/pull/943))
+    - copy enhanced structure files from [GitHub](https://github.com/shopsys/project-base/blob/master/src/Shopsys/ShopBundle/Resources/definition-with-filter/products/) to `src/Shopsys/ShopBundle/Resources/definition-with-filter/products/` folder in your project (create `definition-with-filter` folder)
+    - set elasticsearch structure files directory in `app/config/paths.yml` to `'%shopsys.resources_dir%/definition-with-filter/'`
+    - replace class `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacade` with the interface `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface` in
+        - `src/Shopsys/ShopBundle/Controller/Front/ProductController.php`
+        - `src/Shopsys/ShopBundle/Controller/Front/SearchController.php`
+    - add service definition for facade interface to your `services.yml` like `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface: '@Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainElasticFacade'`
+    - add following alias to `services.yml` and `services_test.yml`
+        ```yaml
+        Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportRepository:
+            alias: Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportWithStructureRepository
+        ```
+    - fix test of export repository `tests/ShopBundle/Functional/Model/Product`
```

Comment:
I would appreciate link with how should the file look like and name/class of the test